src/components/github-htmlview.component.js

```diff
@@ -246,6 +261,33 @@ export class GithubHtmlView extends Component {
         hr: (node, index, siblings, parent, defaultRenderer) => (
           <View key={index} style={{ ...hrStyle }} />
         ),
+        div: (node, index, siblings, parent, defaultRenderer) => {
+          if (node.attribs.class) {
+            const className = node.attribs.class;
+
+            if (className.includes('email-hidden-reply')) {
+              return defaultRenderer(onlyTagsChildren(node), node);
+            }
+
+            if (className.includes('email-quoted-reply')) {
+              return (
+                <ToggleView
+                  TouchableView={<Text style={quotedEmailToggleStyle}>…</Text>}
+                >
+                  {renderers.blockquote(
+                    node,
+                    index,
+                    siblings,
+                    parent,
+                    defaultRenderer
+                  )}
+                </ToggleView>
+              );
+            }
+          }
+
+          return undefined;
```

Comment:
I can not understand why `undefined` instead of `null`?

Reply:
From react-native-htmlview documentation: 
> renderNode: a custom function to render HTML nodes however you see fit. If the function returns undefined (not null), the default renderer will be used for that node.

Reply:
I'm understood, thank you.